docker/cli
# Grace Choi and Pranjal Rai: Solving Issue #923 docker man pages need some love

…exas.edu> Signed-off by: Pranjal Rai <pranjalrai@utexas.edu>



**- What I did**

We changed the three man pages files which included docker-build.1.md, docker-config-json.5.md, and docker-run.1.md. We changed much of the styling based on the guidelines posted on the issue (issue #923) and the accompanying man pages guide. This guide was attached to the issue. We changed much of the phrasing, some bolding, italics, bullet points, naming conventions, spacing, and grammar.

**- How I did it**

We went through each of the requirements posted on the issue page and went through them one by one and changed the man page files accordingly.

**- How to verify it**
You can look at the three files we changed (docker-build.1.md, docker-config-json.5.md, and docker-run.1.md) and either look at the commit history to see how these files have changed or you can run each of these files through a diff checker and compare how the file has changed from the original master copy. 

**- Description for the changelog**

```markdown changelog
Formatting, semantic, and wording changes were made to the man pages.
```

**- A picture of a cute animal (not mandatory but encouraged)**
https://t3.ftcdn.net/jpg/05/85/08/52/360_F_585085296_yrBZ1L2yOlWiLlxS8r4iMPlwytNbPwMl.jpg
